Analyser/src/Coverage.js

```diff
@@ -29,12 +40,22 @@ class Coverage {
             			delete map[j];
             		} else {
                         map[j] = 1;
+
+                        // Convert the sid and instrumented iid into an uninstrumented line number
+                        if (Config.returnUninstrumentedLineCoverage){
+                            let jalangiOutput = this._sandbox.iidToLocation(i + 1, j);
+                            let uninstrumentedLineNumber = lineNumberRegex.exec(jalangiOutput);
```

Comment:
I think instead of using iidToLocation we should pull out the logic that grabs the line numbers from the method rather than pulling it into a string then parsing it back out into a number